requirements.txt

```diff
@@ -17,4 +17,4 @@ six==1.15.0
 SQLAlchemy==1.3.20
 Werkzeug==1.0.1
 WTForms==2.3.3
-psycopg2
\ No newline at end of file
+psycopg2-binary
```

Comment:
I don't think we should switch to the binary version of the library. At least if you think it's necessary/better, we should do this on a separate PR (and testing that nothing breaks with this switch). For now, move this change out of this PR.

Reply:
that wasn't supposed to make it into the commit, sorry